utilities/fzf/fzf.zsh

```diff
@@ -58,10 +58,22 @@ ctrl-u:preview-page-up,ctrl-a:select-all+accept"
 # << list files >>
 # NOTE: mapped to `C-f` instead of default `C-t`
 
-# paste selected files
-bindkey '^f' fzf-file-widget
-export FZF_CTRL_T_COMMAND="$FZF_DEFAULT_COMMAND"
+# Establish ^f to do a file-search in the usual way, that ignores certain files
 export FZF_CTRL_T_OPTS="$_fzf_bat_preview $_fzf_hidden_preview_window"
+function fzf_std_file_widget {
```

Comment:
I don't think there's a reason to define another function here (please let me know what I'm missing if there is a reason) as opposed to binding the existing `fzf-file-widget` to `^f`